src/earn/EarnConfirmationScreen.tsx

```diff
@@ -77,19 +77,18 @@ export default function EarnConfirmationScreen({ route }: Props) {
     [withdrawToken, pool.pricePerShare, inputAmount]
   )
 
-  // Will need this to handle preparing a claim transaction, a withdrawal transaction and a withdrawal and claim transaction
   const {
     result: prepareTransactionsResult,
     loading: isPreparingTransactions,
     error: prepareTransactionError,
-  } = usePrepareWithdrawAndClaimTransactions({
+  } = usePrepareClaimExitAndWithdrawTransactions(mode, {
     amount: withdrawAmountInDepositToken.dividedBy(pool.pricePerShare[0]).toString(),
     pool,
     walletAddress,
     feeCurrencies,
     hooksApiUrl,
     rewardsPositions,
-    useMax,
+    useMax: !!useMax,
```

Comment:
I didn't notice that useMax was boolean | undefined in a previous pr. Can you just make it a boolean type? That would be cleaner.

Reply:
Fixed in e2d92f15d1a4aeab5cdc6bdac95792d010558075!